Add documentation re alphabetical sorted of MV dimensions (#10695)
apache/druid: docs/ingestion/index.md

| type | Either `string`, `long`, `float`, or `double`. | `string` |
| name | The name of the dimension. This will be used as the field name to read from input records, as well as the column name stored in generated segments.<br><br>Note that you can use a [`transformSpec`](#transformspec) if you want to rename columns during ingestion time. | none (required) |
| createBitmapIndex | For `string` typed dimensions, whether or not bitmap indexes should be created for the column in generated segments. Creating a bitmap index requires more storage, but speeds up certain kinds of filtering (especially equality and prefix filtering). Only supported for `string` typed dimensions. | `true` |

> | multiValueHandling | Specify the type of handling for multi-value fields. Possible values are `sorted_array`, `sorted_set`, and `array`. `sorted_array` and `sorted_set` order the array upon ingestion. `sorted_set` removes duplicates. `array` ingests data as-is | `sorted_array` |

Reviewer: It'd be nice to link "multi-value fields" to the `../querying/multi-value-dimensions.md` doc, so these two pages cross-link to each other.

> | multiValueHandling | Specify the type of handling multi-value fields will get, possible values are `SORTED_ARRAY`, `SORTED_SET` and `ARRAY`. The first two will order the array upon ingestion and `SORTED_SET` will remove duplicates. `ARRAY` will ingest data as-is | `SORTED_ARRAY` |

Reviewer 2: ```suggestion
| multiValueHandling | Specify the type of handling for multi-value fields. Possible values are `SORTED_ARRAY`, `SORTED_SET`, and `ARRAY`. `SORTED_ARRAY` and `SORTED_SET` order the array upon ingestion and `SORTED_SET` removes duplicates. `ARRAY` ingests data as-is | `SORTED_ARRAY` |
```
